AsyncDisplayKit/Layout/ASStackLayoutDefines.h

```diff
@@ -85,6 +85,22 @@ typedef NS_ENUM(NSUInteger, ASStackLayoutAlignSelf) {
   ASStackLayoutAlignSelfStretch,
 };
 
+// TODO documentation
+typedef NS_ENUM(NSUInteger, ASStackLayoutFlexWrap) {
+  ASStackLayoutFlexWrapNoWrap,
+  ASStackLayoutFlexWrapWrap,
```

Comment:
I'm not entirely happy with these names. However, I also want to stick to CSS Flexbox spec in which these values are called "wrap" and "no-wrap". Yoga uses the same convention (https://github.com/facebook/yoga/blob/master/yoga/YGEnums.h#L89).